config/initializers/constants.rb

```diff
@@ -0,0 +1,2 @@
+#Place all constants here
+ACCEPTED_FILE_FORMATS = ["image/jpeg","image/png","image/svg+xml","image/gif"]
```

Comment:
IMO, this should be in config/initializers/0_glitterconfig.rb along with the rest of the application configuration options.